osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/periodicmeterreads/GetPeriodicMeterReadsCommandExecutorIntegrationTest.java

```diff
@@ -0,0 +1,537 @@
+    private final ObisCode OBIS_ACTIVE_ENERGY_EXPORT_RATE_2 = new ObisCode("1.0.2.8.2.255");
+
+    private final int CLASS_ID_CLOCK = 8;
+    private final int CLASS_ID_DATA = 1;
+    private final int CLASS_ID_REGISTER = 3;
+    private final int CLASS_ID_PROFILE = 7;
+
+    private final byte ATTR_ID_VALUE = 2;
+    private final byte ATTR_ID_BUFFER = 2;
+    private final byte ATTR_ID_SCALER_UNIT = 3;
+
+    private final DataObject CLOCK = DataObject.newStructureData(
+            Arrays.asList(DataObject.newUInteger16Data(this.CLASS_ID_CLOCK),
+                    DataObject.newOctetStringData(this.OBIS_CLOCK.bytes()),
+                    DataObject.newInteger8Data(this.ATTR_ID_VALUE), DataObject.newUInteger16Data(0)));
+
+    private final DataObject STATUS = DataObject.newStructureData(
+            Arrays.asList(DataObject.newUInteger16Data(this.CLASS_ID_DATA),
+                    DataObject.newOctetStringData(this.OBIS_STATUS.bytes()),
+                    DataObject.newInteger8Data(this.ATTR_ID_VALUE), DataObject.newUInteger16Data(0)));
+
+    private final DataObject ACTIVE_ENERGY_IMPORT_RATE_1 = DataObject.newStructureData(
+            Arrays.asList(DataObject.newUInteger16Data(this.CLASS_ID_REGISTER),
+                    DataObject.newOctetStringData(this.OBIS_ACTIVE_ENERGY_IMPORT_RATE_1.bytes()),
+                    DataObject.newInteger8Data(this.ATTR_ID_VALUE), DataObject.newUInteger16Data(0)));
+
+    private final DataObject ACTIVE_ENERGY_IMPORT_RATE_2 = DataObject.newStructureData(
+            Arrays.asList(DataObject.newUInteger16Data(this.CLASS_ID_REGISTER),
+                    DataObject.newOctetStringData(this.OBIS_ACTIVE_ENERGY_IMPORT_RATE_2.bytes()),
+                    DataObject.newInteger8Data(this.ATTR_ID_VALUE), DataObject.newUInteger16Data(0)));
+
+    private final DataObject ACTIVE_ENERGY_EXPORT_RATE_1 = DataObject.newStructureData(
+            Arrays.asList(DataObject.newUInteger16Data(this.CLASS_ID_REGISTER),
+                    DataObject.newOctetStringData(this.OBIS_ACTIVE_ENERGY_EXPORT_RATE_1.bytes()),
+                    DataObject.newInteger8Data(this.ATTR_ID_VALUE), DataObject.newUInteger16Data(0)));
+
+    private final DataObject ACTIVE_ENERGY_EXPORT_RATE_2 = DataObject.newStructureData(
+            Arrays.asList(DataObject.newUInteger16Data(this.CLASS_ID_REGISTER),
+                    DataObject.newOctetStringData(this.OBIS_ACTIVE_ENERGY_EXPORT_RATE_2.bytes()),
+                    DataObject.newInteger8Data(this.ATTR_ID_VALUE), DataObject.newUInteger16Data(0)));
+
+    private Date TIME_FROM;
+    private Date TIME_TO;
+    private DataObject PERIOD_1_CLOCK;
+    private DataObject PERIOD_2_CLOCK;
+    private Date PERIOD_1_CLOCK_VALUE;
+    private Date PERIOD_2_CLOCK_VALUE;
+    private Date PERIOD_2_CLOCK_VALUE_NULL_DATA_PERIOD_15MIN;
+    private Date PERIOD_2_CLOCK_VALUE_NULL_DATA_PERIOD_DAILY;
+    private Date PERIOD_2_CLOCK_VALUE_NULL_DATA_PERIOD_MONTHLY;
```

Comment:
The above fields are not constants so should be named with camel case, not uppercase-with-underscores

Reply:
you are right, I forgot to rename them after refactoring the test. Also, this should be picked up by Sonar.